src/Tree/AST.hs

```diff
@@ -67,5 +66,8 @@ findByGrammarType :: (Eq g) => g -> AST g a -> Maybe (AST g a)
 findByGrammarType r = findByNode ((r==).grammartype)
 
 -- | Returns all ancestors of a given subtree (second argument) in the given tree (first argument), that are optional (i.e., their 'optionaltype' is @Optional@).
+-- TODO(benjamin) According to the paper `Wrapper` nodes are also `Optional`
+-- The use case of this function is correct thus the name is misleading, given
+-- the definitions in the paper.
```

Comment:
Oh sorry, I didn't actually intend to commit these TODO comments. They are notes which we discussed in one of our meetings. I considered this as resolved after our meeting, because I missed the last part of the documentation comment.